grammars/latex.cson

```diff
@@ -1920,6 +1920,35 @@
       }
     ]
   }
+  {
+    'begin': '((\\\\)(?:(?:gls|GLS|Gls)\\w*\\*?))(?:(\\[)[^\\]]*(\\]))?(\\{)'
```

Comment:
This regex will match anything that starts with `gls`, even including stuff like `gls1234` which is invalid. The scope needs to be heavily reduced, either to a specific list of possible matches, or one like [this](https://github.com/hesstobi/language-latex/blob/2e3aaa68fa9beba51ad058332b85ef16e7ec4119/grammars/latex.cson#L1831)